cli/common.go

```diff
@@ -129,7 +131,32 @@ func createRequest(slices []string) (controller.Request, error) {
 	return req, nil
 }
 
-func printStatus(groupStatus []fleet.UnitStatus) error {
-	fmt.Printf("%#v\n", groupStatus)
+var (
+	statusHeader = "Group | Units | FDState | FCState | SAState | IP | Machine"
```

Comment:
I would prefer having the same column headers as fleetctl

* DState
* State
* Active